lib/doorkeeper/rails/helpers.rb

```diff
@@ -11,11 +11,11 @@ def doorkeeper_authorize!(*scopes)
         end
       end
 
-      def doorkeeper_unauthorized_render_options
+      def doorkeeper_unauthorized_render_options(_error:)
```

Comment:
Oh, I thought the underscore was suggested by houndci and concurred by you in my last commit. Anyway, I'm changing it now.